eslint.config.js

```diff
@@ -0,0 +1,59 @@
+import hypothesis from 'eslint-config-hypothesis';
+import jsxA11y from 'eslint-plugin-jsx-a11y';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+  {
+    ignores: ['lib/**', 'lib-cjs/**', 'build/**'],
```

Comment:
When testing this I found I had a `.yalc` directory locally which was not excluded from linting. It was being used for `@hypothesis/frontend-build`.

Reply:
I have added `.yalc` to the list of ignored content, like in client repo.